pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## What does this PR do?
+
+A short description of what the PR does and motivation for the PR (what problem does it solve? why is this new feature good?)
+
+## Is this related to an open issue?
+
+Include the issue number if so
+
```

Comment:
Should we make this piece mandatory? Aka, if you find a problem, first create the issue, then the PR and tag it, non-optional. In this way the issue will remain open and available while the PR is under review, can still be discused/gather attention

Reply:
i rather see the issues first, i think would be useful for us to see what issues are found, keep PR's contextual, and will also help keep the issues up to date

Reply:
I don't feel too strongly about it. If you feel strongly about it let's add it